src/common/hooks/useScrollToRecommendations.ts

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect } from 'react';
+
+const scrollTo = (div: null | HTMLDivElement, offset: number = 100) =>
+  div &&
+  window.scrollTo({
+    left: 0,
+    top: div.offsetTop - offset,
+    behavior: 'smooth',
+  });
+
+// allowScroll is an optional additional check before scrolling to recommendations. On location pages, we first check to see if there is another param in the url that controls scrolling (ie. a chart share url)
```

Comment:
I'm not really following this.  Would this kick in for a URL like https://covidactnow.org/us/arizona-az/chart/9/recommendations or something?  I feel like I'm missing something.

Okay, looking at the old code a bit more, I think the `scrolledWithUrl` flag was being used to keep us from accidentally re-scrolling to a given section after the page is already loaded (in the case that the component re-renders for some reason and so this hook gets re-run).  I think your PR might be breaking that functionality since we used to do:

```
            setScrolledWithUrl(true);
            scrollTo(recommendationsRef.current);
```

and now we don't call `setScrolledWithUrl(true);` when scrolling to recommendations.  That said, I wasn't able to actually reproduce any bad scrolling behavior, so maybe this isn't an issue anymore.  I guess I'd be a favor of removing `allowScroll` unless we know why it's needed.

Reply:
hmm i went back to the pr where i originally added `scrolledWithUrl` -- i should've added a note as to what it was for...

i'm going to just remove `allowScroll` for now and will revisit a bug pops up. does that sound alright?

Reply:
Yep!